content/_help/specific-agencies/trusted-traveler-programs._fr.md

```diff
@@ -35,13 +35,10 @@ Login.gov ne peut répondre qu'aux questions concernant la création d'un compte
 
 Veuillez contacter directement les [Programmes voyageurs sans risque](https://help.cbp.gov/s/questions?language=en_US) si vous avez des questions concernant:
 
-* Demandes
-  * [État de la demande](https://help.cbp.gov/s/article/Article-1886?language=en_US)
-  * [Modifications de la demande](https://help.cbp.gov/s/article/Article-1377?language=en_US)
-* Rendez-vous
-  * [Prendre ou modifier un rendez-vous](https://help.cbp.gov/s/article/Article-1378?language=en_US)
-  * [Inscription à l’arrivée](https://help.cbp.gov/s/article/Article-1871?language=en_US)
-* [Carte NEXUS, SENTRI ou Global Entry perdue, volée ou endommagée](https://help.cbp.gov/s/article/Article-1206?language=en_US)
+
+* [État de la demande](https://help.cbp.gov/s/article/Article-1886?language=en_US)
+
+* [Inscription à l’arrivée](https://help.cbp.gov/s/article/Article-1871?language=en_US)
```

Comment:
The excess whitespace here is causing inconsistent visual spacing:

![image](https://github.com/GSA-TTS/identity-site/assets/1779930/6e665915-25b8-4955-bd39-df4103111b92)

```suggestion
* [État de la demande](https://help.cbp.gov/s/article/Article-1886?language=en_US)
* [Inscription à l’arrivée](https://help.cbp.gov/s/article/Article-1871?language=en_US)
```